plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/TestRemarksReportingOracleDistributedQueries.java

```diff
@@ -29,7 +32,18 @@ protected QueryRunner createQueryRunner()
             throws Exception
     {
         this.oracleServer = new TestingOracleServer();
-        return OracleQueryRunner.createOracleQueryRunner(oracleServer, ImmutableMap.of(), TpchTable.getTables(), false, true);
+        return OracleQueryRunner.createOracleQueryRunner(
```

Comment:
changes regarding adding new `createOracleQueryRunner` should be extracted to separate commit.